images/win/scripts/Tests/NodeLTS.Tests.ps1

```diff
@@ -0,0 +1,13 @@
+Describe "Node LTS" {
+    It "Checking <ToolName> " -TestCases @(
+        @{ ToolName = "node" }
```

Comment:
I suggest to split this describe with two contexts:
```
Describe "Node.JS" {
  Context "Node" {
    // node and npm here
  }
  Context "Global packages" {
    // all remain packages here
  }
}
```

Reply:
Reworked.

Reply:
Still not resolve. You missed `Context` blocks

Reply:
Thank you.